tests/autoyast/installation.pm

```diff
@@ -173,15 +175,15 @@ sub run {
     $checktime = 30;
     $looptime  = 0;
     $timeout   = 0;
-    while (!$timeout) {
+    while ( !$timeout ) {
 
         mouse_hide(1);
-        $ret = check_screen(["reboot-after-installation", "autoyast-error"], $checktime);
+        $ret = check_screen( [ "reboot-after-installation", "autoyast-error" ], $checktime );
 
         #repeat until timeout or login screen
-        if (defined $ret) {
-            if (match_has_tag('autoyast-error')) {
-                record_soft_failure 'AUTOYAST_EXPECT_ERRORS ' . get_var('AUTOYAST_EXPECT_ERRORS_REASON');
+        if ( defined $ret ) {
+            if ( match_has_tag('autoyast-error') ) {
+                diag( 'Expected error. Iteration = ' . $i );
```

Comment:
Yes you created this PR because the test you found should find fail and should not soft_fail as it was expected to fail and existing test was not written for such test, whoever wrote it.